src/main/java/com/rarchives/ripme/ripper/rippers/ImgurRipper.java

```diff
@@ -108,6 +108,25 @@ public String getAlbumTitle(URL url) throws MalformedURLException {
                 elems = albumDoc.select("meta[property=og:title]");
                 if (elems!=null) {
                     title = elems.attr("content");
+                    logger.debug("Title is " + title);
+                }
+                // This is here encase the album is unnamed, to prevent
+                // Imgur: The most awesome images on the Internet from being added onto the album name
+                if (title.contains("Imgur: The most awesome images on the Internet")) {
```

Comment:
Since we're referencing the default title more than once it should be made into a constant.